doc/developer_guide/conf.py

```diff
@@ -285,7 +285,7 @@
     'Sergi Siso'
 epub_publisher = u'Rupert Ford, Joerg Henrichs, Iva Kavcic, Andrew Porter ' \
     'and Sergi Siso'
-epub_copyright = u'2017-2020, STFC'
+epub_copyright = u'2017-2022, STFC'
```

Comment:
I've done that, and updates all three conf.py files. FWIW, epub_copyright defaults to copyright anyway (according to the doc), and I did not manage to create a document that actually uses epub_copyright (I did create an epub file, and my reader did not find a the `epub_copyright` string anywhere). But since it can't hurt to have this explicit, I left it all in.

Reply:
That's great, thanks.